posthog/api/user.py

```diff
@@ -333,6 +381,17 @@ def request_email_verification(self, request, **kwargs):
 
         return Response({"success": True})
 
+    @action(methods=["POST"], detail=True)
```

Comment:
Being pedantic but REST-wise isn't this more of a PUT operation?

Reply:
Are we creating a choice or updating a choice, it felt like over-kill to have both routes

Think of it as creating the command to apply the choice and POST makes sense 🙃